pkg/advisory/advisory.go

```diff
@@ -197,3 +216,44 @@ func Discover(options DiscoverOptions) error {
 
 	return nil
 }
+
+func determinePackagesToLookup(apkindex *repository.ApkIndex, cfgs *configs.Index) []string {
+	packagesFound := make(map[string]struct{})
+	for _, pkg := range apkindex.Packages {
+		if pkg.Origin == "" {
+			// TODO: find out why this case exists in the real APKINDEX
```

Comment:
I think this was a melange bug we had a while back. Are recent-ish apks showing up this way?

Reply:
No, I haven't seen it in recent apks. And that makes sense! I'll remove the comment.